index.js

```diff
@@ -712,13 +739,16 @@ class BitbucketScm extends Scm {
             token: decodeURIComponent(token)
         };
 
-        return this.breaker.runCommand(options).then(response => {
-            if (response.statusCode !== 201 && response.statusCode !== 200) {
-                throw new Error(`STATUS CODE ${response.statusCode}: ${JSON.stringify(response.body)}`);
+        try {
+            return this.breaker.runCommand(options);
+        } catch (err) {
+            if (err.statusCode !== 201 && err.statusCode !== 200) {
+                logger.error('Failed to getFile: ', err);
+                throw err;
             }
 
-            return response;
-        });
+            return undefined;
```

Comment:
can we return the error for other status codes too?

Reply:
changed to match: https://github.com/screwdriver-cd/scm-github/blob/master/index.js#L1095